packages/core/src/common/_variables.scss

```diff
@@ -117,18 +117,29 @@ $pt-transition-duration: 100ms !default;
 
 // Light theme styles
 
-$pt-input-box-shadow: inset border-shadow(0.15),
-                      inset 0 1px 1px rgba($black, $pt-drop-shadow-opacity) !default;
+$pt-input-box-shadow: inset 0px 1px 1px rgba(17, 20, 24, 0.5),
```

Comment:
Updated default borders for light theme
![Screen Shot 2021-12-01 at 16 36 10](https://user-images.githubusercontent.com/5934779/144317887-730c78ff-3c13-4dc8-8641-beae525d77b0.png)